test/helpers.ts

```diff
@@ -28,10 +34,7 @@ export function stringifyExpectedChanges(expected: [string, object][]): string {
     const obj = update[1] as {[key: string]: string};
     stringified = `${stringified}\n${obj.content.replace(/\r\n/g, '\n')}`;
   }
-  return stringified.replace(
-    /[0-9]{4}-[0-9]{2}-[0-9]{2}/g,
-    '1983-10-10' // use a fake date, so that we don't break daily.
-  );
+  return dateSafe(stringified);
```

Comment:
Lots of usage of `JSON.stringify(expectedChanges, null, 2)` out there instead of `helpers. stringifyExpectedChanges(expectedChanges)` - I wonder if that's intentional or just drift?

Reply:
I think just drift, ideally we'd go through helpers as much as possible.